packages/x-data-grid/src/components/virtualization/GridVirtualScrollbar.tsx

```diff
@@ -69,9 +66,7 @@ const ScrollbarHorizontal = styled(Scrollbar)({
   bottom: '0px',
 });
 
-const Content = styled('div')({
-  display: 'inline-block',
```

Comment:
Not sure why we needed it, but removing it also fixes the issue I just noticed:

Before: https://mui.com/x/react-data-grid/layout/#predefined-dimensions

https://github.com/user-attachments/assets/9c770446-d122-4e45-a0a8-f9c763eef338

After: https://deploy-preview-14614--material-ui-x.netlify.app/x/react-data-grid/layout/#predefined-dimensions

https://github.com/user-attachments/assets/6aa171f6-f81d-4420-a140-f515bdfa6d54